images/stolonctl/entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+# -*- mode: sh; -*-
+
+# File: entrypoint.sh
+# Time-stamp: <2018-03-28 11:51:14>
+# Copyright (C) 2018 Sergei Antipov
+# Description: Entrypoint for PostgreSQL upgarder tool
+
+# set -o xtrace
```

Comment:
is this a debugging left-over or should be useful?

Reply:
It is a part of my default code snippet for bash files